### ISSUE TITLE
Experimental keys on scalar is now forbidden

### ISSUE THREAD

**Issue by eserte** [drzigman/WWW-LogicBoxes#1]
There are compilation errors with perl 5.23.x, showing up when running the test suite:

```
Experimental keys on scalar is now forbidden at /tmpfs/.cpan-build/2016012600/WWW-LogicBoxes-1.0.1-dvFbZU/blib/lib/WWW/LogicBoxes/Role/Command/Raw.pm line 165.
Experimental keys on scalar is now forbidden at /tmpfs/.cpan-build/2016012600/WWW-LogicBoxes-1.0.1-dvFbZU/blib/lib/WWW/LogicBoxes/Domain.pm line 125.
Experimental keys on scalar is now forbidden at /tmpfs/.cpan-build/2016012600/WWW-LogicBoxes-1.0.1-dvFbZU/blib/lib/WWW/LogicBoxes/DomainTransfer.pm line 108.
Experimental keys on scalar is now forbidden at /tmpfs/.cpan-build/2016012600/WWW-LogicBoxes-1.0.1-dvFbZU/blib/lib/WWW/LogicBoxes/Role/Command/Domain/Availability.pm line 50.
```

**Comment by drzigman** [comment]
Thank you (and thanks to all the CPAN Testers) for the heads up about this.  I'm on it and looking to cut a new release today to address this.

**Comment by drzigman** [comment]
Fixed in v1.0.2

Thanks again!
